packages/oas/src/converter/parts/headers/Oas2HeadersConveter.ts

```diff
@@ -17,6 +21,15 @@ export class Oas2HeadersConverter extends HeadersConverter<OpenAPIV2.Document> {
     return this.createHeaders('accept', pathObj.produces);
   }
 
+  protected convertHeaderParam(
+    headerParam: LocationParam<OpenAPIV2.Parameter>
+  ): Header {
+    return {
+      name: headerParam.param.name,
+      value: this.oas2ValueSerializer.serialize(headerParam) as string
+    };
```

Comment:
Use `createHeader()` where it is possible

```suggestion
    return this.createHeader(
      headerParam.param.name,
      this.oas2ValueSerializer.serialize(headerParam) as string
    );
```